webapp/portlet/src/main/webapp/profile-work-experience/components/ProfileWorkExperienceDrawer.vue

```diff
@@ -83,6 +83,14 @@ export default {
           this.handleError(this.$t('profileWorkExperiences.invalidEndDate'));
           return;
         }
+        if (experience.startDate && experience.endDate && new Date(experience.endDate) > new Date()) {
```

Comment:
Why do you need to check the startDate here ?

Reply:
Done!